src/lemlib/logger/buffer.cpp

```diff
@@ -7,7 +7,7 @@
 
 namespace lemlib {
 Buffer::Buffer(std::function<void(const std::string&)> bufferFunc)
-    : bufferFunc(bufferFunc), task([&]() { loggingTask(); }) {}
+    : bufferFunc(bufferFunc), task([&]() { taskLoop(); }) {}
```

Comment:
```suggestion
    : bufferFunc(bufferFunc), task([=]() { taskLoop(); }) {}
```
I don't think that capturing by reference is necessary in this context. This is my bad